.github/workflows/images.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow builds and pushes the latest docker image from the master branch
+name: build
+
+on:
+  workflow_run:
+    workflows: [testsuite]
+    types: [completed]
+
+env:
+  DOCKER_REPO: sheltertechsf
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      - uses: docker/build-push-action@v2
```

Comment:
I think the step to tag the image is still missing, or is that done elsewhere?

Reply:
Ugh, I don't know why I didn't see, right below here it does this 🤦‍♂️

Reply:
Pretty sure this is all handled in this one action, you can see what this pushed to my personal docker hub here: https://hub.docker.com/repository/docker/lexholden/askdarcel-api

There is no version tag, it just pushes `latest`. You're right, we would probably need to support also running this on a release branch with the release name as a tag. Not sure how to do this, can find some time this weekend to figure it out if you think it's a blocker to merging this?

Reply:
https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md

Looks like a good example for this